project/urls.py

```
@@ -14,9 +14,20 @@
 """
+from django.conf import settings
 from django.contrib import admin
-from django.urls import path
+from django.urls import path, include
+from django.conf.urls.static import static
 
 urlpatterns = [
     path('admin/', admin.site.urls),
+    path('', include('home.urls')),
+    path('accounts/', include('accounts.urls')),
+    path('classroom/',include('classrooms.urls'))
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)